chalicelib/app_utils.py

```diff
@@ -19,4 +22,5 @@ class AppUtils(AppUtils_from_core):
     host = HOST
     package_name = 'chalicelib'
     check_setup_dir = dirname(__file__)
-    html_main_title = 'Foursight-CGAP'
+    DEFAULT_ENV = os.environ.get("ENV_NAME", "cgap-uninitialized")
```

Comment:
I think I elsewhere remarked on this, too. I think maybe `'cgap-unknown'` might be better?

Reply:
Yep, made that change.